x-pack/plugins/security_solution/public/detections/components/alerts_table/index.tsx

```diff
@@ -210,9 +210,10 @@ export const AlertsTableComponent: FC<DetectionEngineAlertTableProps> = ({
     return undefined;
   }, [isEventRenderedView]);
 
-  const dataTableStorage = getDataTablesInStorageByIds(storage, [TableId.alertsOnAlertsPage]);
-  const columnsFormStorage = dataTableStorage?.[TableId.alertsOnAlertsPage]?.columns ?? [];
-  const alertColumns = columnsFormStorage.length ? columnsFormStorage : getColumns(license);
```

Comment:
Because of these 3 statements, the reference of `alertColumns` was not stable.  Made the changes accordingly.

Reply:
Thanks!